### PR TITLE
Feat/comm call 2024q4

### PR DESCRIPTION
**Pull request by garloff**

### CODE REVIEW

File: communitycall-data.yml [hunk 1]
```diff
@@ -40,3 +40,8 @@
     "2024-04-25": "mu"
     "2024-08-08": "js"
     "2024-09-19": "fkr"
+    "2024-10-10": "kg"
```
Comment (jschoone) [inline communitycall-data.yml:43]: Already covered by #320, please merge that first

**Review by jschoone — CHANGES_REQUESTED** [review]

**Review by jschoone — APPROVED** [review]

File: communitycall-data.yml [hunk 2]
```diff
@@ -40,3 +40,9 @@
...
+    "2024-10-17": "js"
+    "2024-10-31": "kg"
+    "2024-11-09": "mu"
```
Comment (jschoone) [inline communitycall-data.yml:46]: This is a Saturday

**Review by jschoone — CHANGES_REQUESTED** [review]

File: communitycall-data.yml (hunk 2 above)
Comment (jschoone) [inline communitycall-data.yml:46]:
```suggestion
    "2024-11-07": "mu"
```
As discussed in Daily, this should be 2024-11-07

**Review by jschoone — CHANGES_REQUESTED** [review]

**Review by jschoone — CHANGES_REQUESTED** [review]

**Comment by garloff** [comment]
OK, @jschoone, somehow you need to re-approve as the "changes requested" flag will otherwise block merging.
TIA!

**Review by jschoone — APPROVED** [review]

**Pull request merged**